### PR TITLE
#363 uprage tech stack in the core,hibernate

### LINKED ISSUE #363: Upgrade tech stack in the core, hibernate

### PR DESCRIPTION
**Pull request by Jakuberbetowski**

### CODE REVIEW

File: elibrary-core/pom.xml [hunk 1]
```diff
@@ -107,9 +107,9 @@
 		</dependency>
 		<!-- https://mvnrepository.com/artifact/org.apache.xmlgraphics/batik-transcoder -->
 		<dependency>
-			<groupId>batik</groupId>
+			<groupId>org.apache.xmlgraphics</groupId>
 			<artifactId>batik-transcoder</artifactId>
-			<version>${batik.version}</version>
+			<version>1.16</version>
```
Comment (JaRutkowski) [inline elibrary-core/pom.xml]: IMO it should be put in the brackets (not directly here).

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java [hunk 2]
```diff
@@ -472,7 +472,7 @@ public static BlockReason getByNumber(String blockReason) {
 	public static final Boolean DATA_BASE_REGISTER_DEFAULT_FLAG = false;
 	public static final Integer DATA_BASE_ADMIN_ID = 0;
 	public static final String DATA_BASE_ADMIN_LOGIN = "admin";
-	public static final String DATA_BASE_ADMIN_PASSWORD = "f7e3c24e1a04758097f69be41aa3cf18";
+	public static final String DATA_BASE_ADMIN_PASSWORD = "b86101561d1558c5b2015174070e26b8";
```
Comment (JaRutkowski) [inline elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: Please remove that change 🗑️ .

File: elibrary-hibernate/pom.xml [hunk 3]
```diff
@@ -120,7 +120,7 @@
 		<dependency>
 			<groupId>org.hibernate</groupId>
 			<artifactId>hibernate-core</artifactId>
-			<version>${org.hibernate.version}</version>
+			<version>5.4.10.Final</version>
```
Comment (JaRutkowski) [inline elibrary-hibernate/pom.xml]: We were using parametrized version (in brackets), it should be done in the same this time too.

File: elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateDao.java [hunk 4]
```diff
@@ -5,10 +5,12 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateDao.java]: Please remove all additional new lines.

File: elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateUtil.java [hunk 5]
```diff
@@ -46,13 +48,14 @@ public class HibernateUtil {
...
 
-			Map<String, String> settings = new HashMap<>();
+			Map<String, Object> settings = new HashMap<>();
 			settings.put(Environment.DRIVER, "org.postgresql.Driver");
 			settings.put(Environment.URL, prop.getProperty("db.url"));
 			settings.put(Environment.USER, prop.getProperty("db.username"));
 			settings.put(Environment.PASS, prop.getProperty("db.password"));
 			settings.put(Environment.DIALECT, "org.hibernate.dialect.PostgreSQL9Dialect");
-			settings.put(Environment.CACHE_PROVIDER_CONFIG, "org.hibernate.cache.internal.NoCacheProvider");
+			//TODO: property commented after upgrade to Hibernate 6
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateUtil.java]: Please remove that change 🗑️ .

File: elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateUtil.java [hunk 6]
```diff
@@ -46,13 +48,14 @@ public class HibernateUtil {
 
 			StandardServiceRegistryBuilder registryBuilder = new StandardServiceRegistryBuilder();
 
-			Map<String, String> settings = new HashMap<>();
+			Map<String, Object> settings = new HashMap<>();
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateUtil.java]: Please remove that change 🗑️ .

File: elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/common/Common.java [hunk 7]
```diff
@@ -21,6 +16,10 @@
 import com.javafee.elibrary.hibernate.dto.library.Client;
 import com.javafee.elibrary.hibernate.dto.library.LibraryData;
 
+import javax.persistence.TypedQuery;
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/common/Common.java]: Please put all the imports in the places where they were. 
It's related to the all changed files.

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
Please see the comments and please take care not to add any changes, which don't have any meaning like additional new lines or changes or just changing the location of the code's lines in the files (it's related to the imports).

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java (hunk 2 above)
Comment (Jakuberbetowski) [reply elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: Done

File: elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateDao.java (hunk 4 above)
Comment (Jakuberbetowski) [reply elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/HibernateDao.java]: Done

**Pull request merged**